src/task/script.js

```diff
@@ -206,8 +206,10 @@ taskForm.onsubmit = async (e) => {
 
     const result = await response.json();
 
+    if (response.ok) {
+      window.location.reload(true);
+    }
     alert(result.message);
```

Comment:
What if the task was created successfully?
Will the user get a response message shown?

Reply:
Yes, Only reload has been avoided. Users receive messages after successful task creation.

Reply:
No, I don't think so 
Here line 210 will be executed if the response is successful which reloads the page so line 212 will not be shown to the user.
Please fix it